pkg/filesystem/filesystem.go

```diff
@@ -151,13 +153,28 @@ func mountRootfs(ipList []string, target string, cluster *v2.Cluster, initFlag b
 		common.DefaultTheClusterRootfsDir(cluster.Name),
 		runtime.GetMaster0Ip(cluster))
 	src := common.DefaultMountCloudImageDir(cluster.Name)
+	renderEtc := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.EtcDir)
+	renderChart := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.RenderChartsDir)
+	renderManifests := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.RenderManifestsDir)
 	// TODO scp sdk has change file mod bug
 	initCmd := fmt.Sprintf(RemoteChmod, target)
 	envProcessor := env.NewEnvProcessor(cluster)
 	for _, IP := range ipList {
 		wg.Add(1)
 		go func(ip string) {
 			defer wg.Done()
+			g := new(errgroup.Group)
+			g.Go(func() error {
```

Comment:
Go calls the given function in a new goroutine.
The first call to return a non-nil error cancels the group; its error will be returned by Wait.

'errgroup.group. Go(func() error)' is already a goroutine